pkg/uroot/source.go

```diff
@@ -69,7 +69,7 @@ func buildToolchain(opts BuildOpts) error {
 	}
 
 	toolDir := filepath.Join(opts.TempDir, fmt.Sprintf("go/pkg/tool/%v_%v", opts.Env.GOOS, opts.Env.GOARCH))
-	for _, pkg := range []string{"compile", "link", "asm"} {
+	for _, pkg := range []string{"pack", "compile", "link", "asm"} {
```

Comment:
pack necessary? the tool doesn't build with -x -a by default, i think. if so, let's have that in a different commit.

Reply:
This is a workaround that goes away in 1.10, we hope, but yeah I can move it to a separate commit. 

That said, I think when we demo this thing, it's nice to have the output again for a while. It really test the point across that we're compiling. 

For now, I will remove pack, set the verbose back to false, and that will clean this up.